cdc/entry/schema_storage.go

```diff
@@ -223,13 +227,21 @@ func (s *schemaSnapshot) Clone() *schemaSnapshot {
 
 	tables := make(map[int64]*model.TableInfo, len(s.tables))
 	for k, v := range s.tables {
-		tables[k] = v.Clone()
+		tables[k] = v
```

Comment:
Is it safe to change L224 to `schemas[k] = v`.

The behavior is to change `TableInfo` when applying DDL changes, better to leave some comments here about the principle and comments the function is not a totally `Clone`.